<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtmosPherePage</class>
 <widget class="QWidget" name="AtmosPherePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>471</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>基本设置</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>430</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,0,0">
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_23">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="option_name" stdset="0">
        <string>gas_file</string>
       </property>
       <item>
        <property name="text">
         <string>us-standard</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>subarctic_winter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>subarctic_summer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>midlatitude_summer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>midlatitude_winter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tropics</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_67">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>廓线自定义</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_65">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>吸收波段模型</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_pressure">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>值</string>
       </property>
       <property name="option_name" stdset="0">
        <string>pressure</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_29">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="option_name" stdset="0">
        <string>pressure_file</string>
       </property>
       <item>
        <property name="text">
         <string>us-standard</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>subarctic_winter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>subarctic_summer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>midlatitude_summer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>midlatitude_winter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tropics</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_atmos_define">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>文件路径</string>
       </property>
       <property name="option_name" stdset="0">
        <string>atmosphere_define</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_31">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>气压</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="comboBox_31">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="option_name" stdset="0">
        <string>mol_abs_param_type</string>
       </property>
       <item>
        <property name="text">
         <string>off</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kato</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kato2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kato2.96</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kato2andwandji</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>fu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>avhrr_kratz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sbdart</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>lowtran</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>reptran</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>reptran_channel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>crs</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_68">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>导入模型</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_61">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>温度廓线</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_32">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>气体分子廓线</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_28">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="option_name" stdset="0">
        <string>latitude_file</string>
       </property>
       <item>
        <property name="text">
         <string>us-standard</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>subarctic_winter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>subarctic_summer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>midlatitude_summer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>midlatitude_winter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tropics</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_66">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>湿度廓线</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_4">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>ck_reptran_arg</string>
       </property>
       <property name="option_name" stdset="0">
        <string>mol_abs_param_arg</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_69">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>气压廓线</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_27">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="option_name" stdset="0">
        <string>temperature_file</string>
       </property>
       <item>
        <property name="text">
         <string>us-standard</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>subarctic_winter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>subarctic_summer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>midlatitude_summer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>midlatitude_winter</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tropics</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_atmos_define_clear">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_mixing_ratio_clear">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设置多行</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_33">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>混合比</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_mixing_ratio">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>值</string>
       </property>
       <property name="option_name" stdset="0">
        <string>mixing_ratio</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_70">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>分子柱体含量</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_mol_modify">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>值</string>
       </property>
       <property name="option_name" stdset="0">
        <string>mol_modify</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设置多行</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_mol_modify_clear">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
